recipes/rpy2-arrow/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "rpy2-arrow" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7c627a733f63ca663ffb5d3290097dd8bab83384fa60620228ff68bd04d5b26a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - rpy2 >=3.5
+    - pyarrow >=5
+    - r-arrow >=5
+
+test:
+  imports:
+    - rpy2_arrow.pyarrow_rarrow  # [not win]  # R environment in conda build not correctly configured, but will be at runtime by the user.
```

```suggestion
    - rpy2_arrow.pyarrow_rarrow
```
You can remove this. The feedstock will only build in linux.

---

Oh, interesting. I added that in order to get the builds to pass in the staging repo. I'll just remove the commit.